app/client/styles/main.scss

```diff
@@ -13,10 +13,9 @@
 
 // 3. Themes
 @import
-  // 'themes/default',
-  // 'themes/june13';
-  // 'themes/bauhaus';
-  'themes/tst';
+// symlink your theme file to "themes/_active.scss" and it will override the default
+// if you don't have a theme, just `ln -s _default.scss _active.scss`
+  'active';
```

Comment:
Wouldn't this need to be 'themes/active'?

Reply:
no, the change to the webpack config makes the prefix unnecessary. instead, we resolve `themes/_active.scss` preferentially over `themes/default/_active.scss`